hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -271,24 +245,30 @@ public void persistToken(Token token) {
 		mapModelChangesToMutable(token, mutableToken);
 
 		if (token.hasMintedUniqueTokens()) {
-			final var currentNfts = nfts.get();
+			final var currentNfts = uniqueTokens.get();
+			final var currentUniqueTokenAssociations = uniqueTokenAssociations.get();
+			final var currentUniqueTokenAccountOwnerships = uniqueOwnership.get();
 			for (var uniqueToken : token.mintedUniqueTokens()) {
 				final var mintKey = new MerkleUniqueTokenId(
```

Comment:
nit: lets rename `mintKey` to `merkleUniqueTokenId`. Someone might confuse `mintKey` for a cryptographic key. Same thing applies to `burnKey`